README.md

```diff
@@ -106,3 +106,29 @@ In summary:
 
 ---
 
+
+## Using Archetypes
+
+Archetypes support is part of Hugo, and is used when a user wants to create new content.
+Archetypes make that whole process easier by creating a file (or leaf bundle) with all the mandatory and optional front matter values automatically populated where possible, or otherwise to have place holder content, that the user could then go and update for their respective content needs.
+
+The `hugo new content` command creates a new file in the content directory, using an archetype as a template. 
+For example, the following command would use the default template:
+```
+hugo new content posts/my-first-post.md
```

Comment:
Hugo assumes content directory since that's how Hugo is structured

Reply:
Since all Hugo content is only allowed inside the `content` directory, therefore the `new content` cmd creates the new content bundle inside the content folder automatically, so the path is relative to the `content` dir.

https://gohugo.io/commands/hugo_new_content/